compiler/src/main/scala/org/scalajs/core/compiler/JSEncoding.scala

```diff
@@ -86,6 +86,12 @@ trait JSEncoding extends SubComponent { self: GenJSCode =>
         refClass.values ++ volatileRefClass.values)
   }
 
+  /** See comment in `encodeFieldSym()`. */
+  private lazy val applyFixFor2382 = {
```

Comment:
Could we give this a more descriptive name? As in "mangleOuterPointerName" or something?